### DISCUSSION TITLE
Laravel Session Issue

### DISCUSSION THREAD

**Discussion by nouphaltklm** [laravel/framework#34433] [General]
- Laravel Version: 6.18.35
- PHP Version: 7.2.28
- Database Driver & Version: MySQL 5.7.27

### Description:
Session is not persisting. I have tried all the three option $request->session(), session() (helper) and Illuminate\Support\Facades\Session but all of these fails unless we use save() method. But the use of save method is not mentioned in official document. 

### Steps To Reproduce:
Its a fresh installation: 
eg: 
Setting route : session()->put('mySession','Bob');
Get Route: echo session()->get('mySession'); (Empty)

Working code below:
 Setting route :
 session()->put('mySession','Bob');
 session()->save();
Get Route: echo session()->get('mySession'); (Bob)

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. 

You can post support questions on our discussions tab instead so I'm moving it over there. I won't be able to help you out personally but others who are watching this repository can help you out.

Alternatively, you can post this question on one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

Thanks.
